fem/gridfunc.hpp

```diff
@@ -345,14 +345,32 @@ class GridFunction : public Vector
        projection matrix. */
    void ProjectGridFunction(const GridFunction &src);
 
+   /** @brief Project @a coeff Coefficient element-wise to @a this GridFunction.
+       The projection computation used on each element depends on the choice of
+       FiniteElementSpace #fes. Note that this is not generally an orthogonal
+       projection but may be an element-wise interpolation at element
+       integration points. */
    virtual void ProjectCoefficient(Coefficient &coeff);
 
+   /** @brief Project @a coeff Coefficient element-wise to @a this GridFunction,
```

Comment:
It was not clear to me what this method does.  It might be useful to replace `element-wise` with `dof-wise` or something similar.  Perhaps it should also be stated that only degrees of freedom listed in the `dofs` array will be altered all others will remain unchanged.